lib/include/hse_ikvdb/kvdb_rparams.h

```diff
@@ -101,7 +101,7 @@ struct kvdb_rparams {
 
     uint32_t keylock_tables;
 
-    bool   dio_disable[HSE_MCLASS_COUNT];
+    bool   dio_enable[HSE_MCLASS_COUNT];
```

Comment:
Last nit...dio_enabled?

Reply:
Right, I had the same thought. Then I saw perfc_enable, dur_enable etc. in this struct and went with dio_enable.

Reply:
Actually, let me change it to dio_enabled. That sounds correct.

Reply:
leaving this unchanged for now...